ocis-pkg/config/envdecode/envdecode_test.go

```diff
@@ -58,7 +58,7 @@ type testConfig struct {
 	UnmarshalerNumber unmarshalerNumber `env:"TEST_UNMARSHALER_NUMBER"`
 
 	DefaultInt      int           `env:"TEST_UNSET,asdf=asdf,default=1234"`
-	DefaultSliceInt []int         `env:"TEST_UNSET,asdf=asdf,default=1;2;3"`
+	DefaultSliceInt []int         `env:"TEST_UNSET,asdf=asdf,default=1"`
```

Comment:
```suggestion
	DefaultSliceInt []int         `env:"TEST_UNSET,asdf=asdf,default=1,2,3"`
```

Reply:
that doesn't work. You cannot use comma as a separator in the tag annotation 🤣

Reply:
That doesn't work here as `,` is used to separate `TEST_UNSET`, `asdf=asdf` and `default=1,2,3`